pages/docs/reference/whatsnew1430.md

```diff
@@ -0,0 +1,473 @@
+|`String.toLowerCase()`|`String.lowercase()`|
+|`String.capitalize()`|`String.replaceFirstChar { it.uppercase() }`|
+|`String.decapitalize()`|`String.replaceFirstChar { it.lowercase() }`|
+
+* For `Char` functions:
+
+|**Earlier versions**|**1.4.30 alternative**| 
+| --- | --- |
+|`Char.toUpperCase()`|`Char.uppercaseChar(): Char`<br/>`Char.uppercase(): String`|
+|`Char.toLowerCase()`|`Char.lowercaseChar(): Char`<br/>`Char.lowercase(): String`|
+|`Char.toTitleCase()`|`Char.titlecaseChar(): Char`<br/>`Char.titlecase(): String`|
+
+> For Kotlin/JVM, there are also overloaded `uppercase()`, `lowercase()`, and `titlecase()` functions with an explicit `Locale` parameter.
+{:.note}
+
+See the full list of changes to the text processing functions in [KEEP](https://github.com/Kotlin/KEEP/blob/master/proposals/stdlib/locale-agnostic-string-conversions.md).
+
+### Clear Char-to-code and Char-to-digit conversions
+
+> The unambiguous API for the `Char` conversion feature is [Experimental](evolution/components-stability.html). It may be dropped or changed at any time.
+> Use it only for evaluation purposes.
+> We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-44333).
+{:.note}
+
+The current `Char` to numbers conversion functions, which return UTF-16 codes expressed in different numeric types,
+are often confused with the similar String-to-Int conversion, which returns the numeric value of a string:
+
+<div class="sample" markdown="1" mode="kotlin" theme="idea" data-highlight-only>
+
+```kotlin
+"4".toInt() // returns 4
+'4'.toInt() // returns 52
+// and there was no common function that would return the numeric value 4 for Char '4'
+```
+</div>
+
+To avoid this confusion we've decided to separate `Char` conversions into two following sets of clearly named functions:
+
+* Functions to get the integer code of `Char` and to construct `Char` from the given code:
+
+<div class="sample" markdown="1" mode="kotlin" theme="idea" data-highlight-only>
+
+```kotlin
+fun Char(code: Int): Char
+fun Char(code: UShort): Char
+val Char.code: Int
+```
+</div>
+
+* Functions to convert `Char` to the numeric value of the digit it represents:
+
+<div class="sample" markdown="1" mode="kotlin" theme="idea" data-highlight-only>
+
+```kotlin
+fun Char.digitToInt(radix: Int): Int
+fun Char.digitToIntOrNull(radix: Int): Int?
+```
+</div>
+
+* An extension function for `Int` to convert the non-negative single digit it represents to the corresponding `Char` representation:
+
+<div class="sample" markdown="1" mode="kotlin" theme="idea" data-highlight-only>
+
+```kotlin
+fun Int.digitToChar(radix: Int): Char
+```
+</div>
+
+See more details in [KEEP](https://github.com/Kotlin/KEEP/blob/master/proposals/stdlib/char-int-conversions.md).
+
+## Serialization updates
+
+Along with Kotlin 1.4.30, we are releasing `kotlinx.serialization` [1.1.0](https://github.com/Kotlin/kotlinx.serialization/releases/tag/v1.1.0),
```

Comment:
Do we release 1.1.0 with 1.4.30? @qwwdfsad wrote about 1.1.0-RC a few days ago.

Reply:
@sandwwraith told me that these features will be released in 1.1.0

Reply:
Seems like it's 1.0.0-RC actually.